tests/timeseries/test_base_types.py

```diff
@@ -0,0 +1,79 @@
+# License: MIT
+# Copyright © 2023 Frequenz Energy-as-a-Service GmbH
+
+"""Tests for battery pool."""
+
+
+from datetime import datetime
+
+from frequenz.sdk.timeseries._base_types import Bounds, SystemBounds
+from frequenz.sdk.timeseries._quantities import Power
+
+
+def test_bounds_contains() -> None:
+    """
+    Test the `__contains__` method of the `Bounds` class.
+
+    This method checks if a value is within the defined bounds.
+    """
```

Comment:
Also nitpick: We usually use this format, I'm actually surprised is not enforced by the linters, I thought I received linting errors because of this in the past, now I wonder if I'm wrong or newer versions relaxed this requirement.
```suggestion
    """Test the `__contains__` method of the `Bounds` class.

    This method checks if a value is within the defined bounds.
    """
```